src/Sarif.Converters/Extensions.cs

```diff
@@ -9,6 +9,12 @@ namespace Microsoft.CodeAnalysis.Sarif.Converters
 {
     public static class Extensions
     {
+        // Compare tool format strings with appropriate comparison type.
+        public static bool MatchesToolFormat(this string toolFormat, string other)
+        {
+            return toolFormat.Equals(other, StringComparison.OrdinalIgnoreCase);
```

Comment:
>StringComparison.OrdinalIgnoreCase [](start = 44, length = 34)

well, fine, this is ok. but we probably should add a test that ensures no string declared in ToolFormat differs from another only by case.